plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/ObjectIdFunctions.java

```diff
@@ -60,7 +60,8 @@ public static Slice objectid(@SqlType(StandardTypes.VARCHAR) Slice value)
     public static long objectidTimestamp(@SqlType("ObjectId") Slice value)
     {
         int epochSeconds = new ObjectId(value.getBytes()).getTimestamp();
-        return packDateTimeWithZone(SECONDS.toMillis(epochSeconds), UTC_KEY);
+        long result = packDateTimeWithZone(SECONDS.toMillis(epochSeconds), UTC_KEY);
+        return result;
```

Comment:
Revert.

Reply:
If i revert that the tests will fail. The methods use to create the objectid has changed. This correction is what allows the test to pass and return the expected fail.

Reply:
Which tests are you mentioning?

Reply:
Oh i'm sorry i didn't notice i forgot to revert that.